_includes/filezilla.md

```diff
@@ -0,0 +1,27 @@
+## General instructions
+
+The raw data of all studies published in IDR can be downloaded using the FTP. We recommend using the [FileZilla client](https://filezilla-project.org/download.php).
```

Comment:
Maybe `using anonymous File Transfer Protocol (FTP)` 

This would also explain why there is no user/password below

Reply:
fixed in https://github.com/IDR/idr.openmicroscopy.org/pull/189/commits/1d0c818608e0e647abf7ca0b8a2f8a5f39b21abf